ARCs/arc-0071.md

```diff
@@ -0,0 +1,219 @@
+
+```json
+{
+  "standard": "arc71",
+  "Owner":"NPZIAIWWTFEB5BN6ISEH3JT5AAEDJWKHA3AR7LVY3CO5CBAZFCYHBKECQE", 
+  "Issuer":"NPZIAIWWTFEB5BN6ISEH3JT5AAEDJWKHA3AR7LVY3CO5CBAZFCYHBKSADS", 
+  "description": "proof of participation of defi meeting",
+  "external_url": "https://bitcoinevents.co.za/wp-content/uploads/2022/02/DEFI-AD-2022-1.jpg",
+  "mime_type": "image/jpeg",
+  "properties": {
+    "Topic":"Defi 2022",
+    "Duration": "2021-9-11-2021-9-13",
+    "identity":"speaker"
+  }
+}
+```
+
+An example of possible ASA parameters would be:
+
+* *Asset Name*: `Defi SBT` for example.
+* *Unit Name*: `SBT` for example.
+* *Asset URL*: `ipfs://bafkreigfvngoydofemwj5x5ioqsaqarvlprzgxinkcv3am3jpv2sysqobi`
+* *Metadata Hash*: the 32 bytes of the SHA-256 digest of the soulbound data file.
+* *Total Number of Units*: 1
+* *Number of Digits after the Decimal Point*: 0
+
+
+#### Mutability
+
+##### Rendering
+
+Clients **SHOULD** render an ASA's latest ARC-71 metadata. Clients **MAY** render an ASA's previous ARC-71 metadata for changelogs or other historical features.
+
+##### Updating metadata
+
+If an SBT has a manager address, then the manager **MAY** update an SBT's ARC-71 metadata. To do so, the manager sends a new `acfg` transaction with the entire metadata represented as JSON in the transaction's `note` field.
+
+##### Making metadata immutable and removing the freeze manage account to make it non-transferable
+
+Managers **SHOULD** make an SBT's ARC-71 immutable and non-transferable. To do so, they MUST remove the SBT's manager address and the SBT's freeze address with an `acfg` transaction.
+
+##### About AssetCloseTo `aclose` transaction
+
+An `aclose` transaction sends all of an ASA away and opts out of it. Thus, SBT also faces the challenge that it may be sent to its issuer/creator by `aclose` transaction. In order to solve this, a couple of methods could be applied:
+
+1. Smart contract management: all SBTs **MAY** be issued and maintained by smart contracts which are publicly-verifiable, the smart contracts might be deployed by issuers/SBT creators. For example, a defi meeting organizer may mint all SBTs, whose owners are those who have participated in the meeting, and send them to a smart contract which is used for the purpose of maintaining the SBT. Therefore, all third parties could verify the information contained in the SBT held by the smart contract. 
+
+2. New specifications on Algorand Wallet Transaction Signing API (defined in [ARC-0001](./arc-0001)): In _Transaction Validation_ - _Inputs that Must Be Systematically Rejected_: Transactions WalletTransaction.txn with field txn.AssetCloseTo **MUST** be rejected. 
+
+As ARC-71 has specified the issuer and owner at the time it is minted, it might be not so important for owners to hold the asset, In other words, third parties could retrieve a soul (Algorand accounts) SBT via the SBT metadata on-chain. 
```

Comment:
I am not a fan of this approach to looking up indexers for acfg txn. It can be messy really fast.